src/App.tsx

```diff
@@ -1,39 +1,31 @@
 import { useEffect, useState } from 'react'
 import { useTranslation, Trans } from 'react-i18next'
-import fetch from 'cross-fetch'
 
 import reactLogo from './assets/react.svg'
 import shieldjsLogo from './assets/shieldjs.svg'
 import './App.css'
 import { Button } from './components/Button'
 import { Link } from 'react-router-dom'
-
-interface User {
-  id: number;
-  name: string;
-  username: string;
-}
+import { getCharacter } from './api/rickApi/rickApi'
```

Comment:
Mmm, `/rickApi/rickApi`... yes, I know it's because new folder structure,  but I don't like it... a debate again 😅

Reply:
https://media.giphy.com/media/MRsWGk355jTJXeQfvf/giphy.gif